android/src/main/java/com/reactnativecommunity/imageeditor/ImageEditorModule.java

```diff
@@ -170,6 +170,7 @@ public boolean accept(File dir, String filename) {
   public void cropImage(
       String uri,
       ReadableMap options,
+      boolean useExternalCache,
```

Comment:
as we're already passing a `ReadableMap options,` above, let's include this flag in there to avoid changing the function signature (which would be a breaking change